src/sage/groups/additive_abelian/additive_abelian_group.py

```diff
@@ -225,6 +226,7 @@ def __init__(self, cover, relations):
             True
         """
         FGP_Module_class.__init__(self, cover, relations)
+        Parent.__init__(self, category=CommutativeAdditiveGroups())
```

Comment:
Does this not initialize `Parent` twice? Should `FGP_Module_class` get a `category` argument?